src/main/java/com/google/sps/data/Utility.java

```diff
@@ -38,11 +34,13 @@ public static GraphNode protoNodeToGraphNode(Node thisNode) {
     return GraphNode.create(thisNode.getName(), newTokenList, newMetadata);
   }
 
-  /*
-   * Converts a Guava graph into a String encoding of a JSON Object. The object
-   * contains nodes, edges, and the roots of the graph.
+  /**
+   * Converts a Guava graph into a String encoding of a JSON Object. The object contains nodes,
+   * edges, and the roots of the graph.
    *
    * @param graph the graph to convert into a JSON String
+   * @param roots the roots of the graph to convert into a JSON String
+   * @return a JSON object containing as entries the nodes, edges and roots of this graph
    */
   public static String graphToJson(MutableGraph<GraphNode> graph, HashSet<String> roots) {
```

Comment:
Put this in DataGraph as well? Because you can just get the graph and the roots from the DataGraph object